README.md

```diff
@@ -85,6 +85,47 @@ kubectl create -f deploy/pheweb-deployment-r6.yaml
 
 More [here](https://kubernetes.io/docs/reference/kubectl/cheatsheet/)
 
+
+# Adding external data to be shown together with FINNGEN phenotype results (UKBB)
+
+## Prepare sumstats
+
+Prepare external summary stat files for each phenotype (tabix indexed, same chr build as FinnGen results) and place them in NFS mount. Currently required column names are: ["achr38","apos38","REF","ALT","beta","pval"] the default after lifting over FinnGen data.
+
+Create a manifest for all of the sumstats and what FinnGen phenotype each matches to.
+
+required columns:
+NAME: matching FinnGen phenotype name
+ncases: number of cases (numeric)
+ncontrols:  number of controls (numeric)
+file: full path to the tabixed summary stats.
+
```

Comment:
-All of the lines should be reminated with 2 spaces, so they are on newlines. Alternatively use a list
```markdown
required columns:
- NAME: matching FinnGen phenotype name
- ncases: number of cases (numeric)
- ncontrols:  number of controls (numeric)
- file: full path to the tabixed summary stats.
```

The list also does not have freeform pheno description, which is necessaryun external_matrix.

Reply:
added and also clarified that no header needed and the order of column matters